server/src/OnDeviceComponent.spec.ts

```diff
@@ -651,7 +670,7 @@ describe('OnDeviceComponent', function () {
 		it('should succeed if given a valid node for its parent keyPath and should return timeTaken value', async () => {
 			const args = {keyPath: 'AuthManager.isLoggedIn'};
 			await setAndVerifyValue({...args, value: false});
-			const observePromise = odc.observeField(args);
+			const observePromise = odc.observeField({...args});
```

Comment:
Why are you cloning the object here? If it's a concern about not morphing the outside data, would it make more sense for `observeField` to clone the data on the inside instead? Although perhaps the clone is just to help with the unit test?